gnosis/cowswap
# 1268/4 decimals for amounts everywhere

# Summary

Closes #1268 

Makes all displayed amounts up to at most 4 decimals of precision.
Any amount bellow that will show `< 0.0001`.

USD estimations and percentages remain at 2 decimals.
Prices remain at 6 decimals.

  # To Test

1. Fill in all values on the input
* Observe all amounts have at most 4 decimals, including fee breakdowns
2. Click on swap
* Observe confirmation modal has 4 decimals in all currency amounts
3. Place order
* Observe toast notification has 4 decimals
4. While on expert mode, same should be true
